draft_fifa/context_processors.py

```diff
@@ -1,7 +1,8 @@
+from django.core.handlers.wsgi import WSGIRequest
 from fifa_draft.models import Group
 
 
-def pick_alert(request):
+def pick_alert(request: WSGIRequest) -> dict:
```

Comment:
W przypadku dicta, dobrze otypować tez jego wnętrze.

W przypadku Pythona 3.9+:
`dict[typ_klucza, typ_wartości]`
np.
`dict[str, int]`

W przypadku wcześniejszych wersji Pythona:
```
from typing import Dict

Dict[typ_klucza, typ_wartości]
```

Może się też zdarzyć, że np. wartości w słowniku mogą być różnych typów, np. `int` albo `str`, a klucz np. tylko `str`. Wtedy wyglądałoby to tak:

Python 3.10:
`dict[str, int | str]`

Python 3.9:
```
from typing import Union

dict[str, Union[int, str]]
```

Starsze Pythony:
```
from typing import Dict, Union


Dict[str, Union[int, str]]
```